src/lib/components/Menu/Menu.js

```diff
@@ -298,6 +300,34 @@ class Menu extends Component {
     };
   }
 
+  searchResultClick() {
```

Comment:
Can we add doc strings for these methods?

Reply:
Added